github-linguist.gemspec

```diff
@@ -16,7 +16,7 @@ Gem::Specification.new do |s|
   s.add_dependency 'charlock_holmes', '~> 0.7.3'
   s.add_dependency 'escape_utils',    '~> 1.1.0'
   s.add_dependency 'mime-types',      '>= 1.19'
-  s.add_dependency 'rugged',          '0.26.0b3'
```

Comment:
Loosening this version is fine provided it remains compatible with dotcom (this was a problem in previous releases).

However, can you confirm your branch here is build from the latest? We actually bumped this a while ago and it looks like you were building off something old here.

https://github.com/github/linguist/blob/master/github-linguist.gemspec#L19